spec/sample_feeds/itunes.xml

```diff
@@ -20,14 +20,22 @@
 <itunes:email>john.doe@example.com</itunes:email>
 </itunes:owner>
 <itunes:image href="http://example.com/podcasts/everything/AllAboutEverything.jpg" />
-  <itunes:category text="Technology">
-    <itunes:category text="Gadgets"/>
-  </itunes:category>
-  <itunes:category text="TV &amp; Film"/>
-  <itunes:category text="Arts">
-    <itunes:category text="Design"/>
-    <itunes:category text="Food"/>
-  </itunes:category>
+<image>
+  <url>https://tenderlovemaking.com/images/header-logo-text-trimmed.png</url>
+  <title>Tender Lovemaking</title>
+  <link>http://tenderlovemaking.com</link>
+  <width>766</width>
+  <height>138</height>
+  <description>The act of making love, tenderly.</description>
+</image>
```

Comment:
I specifically added this feature because I found a podcast RSS feed with an `<image>` but no `<itunes:image>`: http://feeds.theincomparable.com/best

```xml
<rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:sy="http://purl.org/rss/1.0/modules/syndication/" xmlns:admin="http://webns.net/mvcb/" xmlns:atom="http://www.w3.org/2005/Atom/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd" xmlns:feedpress="https://feed.press/xmlns" version="2.0">
  <channel>
    <feedpress:locale>en</feedpress:locale>
    <atom:link rel="self" href="http://feeds.theincomparable.com/best"/>
    <atom:link rel="hub" href="http://feedpress.superfeedr.com/"/>
    <image>
      <link>https://www.theincomparable.com/jsnell/</link>
      <title><![CDATA[The Incomparable's Greatest Hits]]></title>
      <url>http://static.feedpress.it/logo/holiday.jpg</url>
    </image>
    <title>The Incomparable's Greatest Hits</title>
    <link>https://www.theincomparable.com/jsnell/</link>
    <language>en-us</language>
    <copyright>Copyright © The Incomparable</copyright>
    <description>Classic episodes of The Incomparable, delivered weekly.</description>
    <itunes:author>The Incomparable</itunes:author>
    <category>TV &amp; Film</category>
    <itunes:summary> Classic episodes of The Incomparable.</itunes:summary>
    <itunes:explicit>no</itunes:explicit>
    <itunes:owner>
      <itunes:name>Jason Snell</itunes:name>
      <itunes:email>jsnell@intertext.com</itunes:email>
    </itunes:owner>
    <itunes:category text="TV &amp; Film"/>
    <item>
    …
```